AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -97,74 +182,299 @@ def AtomEditorComponents_Light_AddedToEntity():
+                    light_component.get_component_property_value(
+                        AtomComponentProperties.light('Shadows Bias')) == 100)
+
+                # 15. Edit the Normal shadow bias parameter.
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Normal shadow bias'), 10)
+                Report.result(
+                    Tests.edit_normal_bias,
+                    light_component.get_component_property_value(
+                        AtomComponentProperties.light('Normal shadow bias')) == 10)
+
+                # 16. Set the Shadowmap size.
+                # for shadowmap_size in SHADOWMAP_SIZE.keys():
+                #     light_component.set_component_property_value(
+                #         AtomComponentProperties.light('Shadowmap size'), SHADOWMAP_SIZE[shadowmap_size])
+                #     current_shadowmap_size = light_component.get_component_property_value(
+                #         AtomComponentProperties.light('Shadowmap size'))
+                #     test_shadowmap_size = (
+                #         f"Shadowmap size for Light type: {light_type} set to {shadowmap_size}.",
+                #         f"P1: Shadowmap size for Light type: {light_type} failed to be set to {shadowmap_size}.")
+                #     Report.result(test_shadowmap_size, current_shadowmap_size == SHADOWMAP_SIZE[shadowmap_size])
+
+                # Shadow filter method tests.
+                # 17. Set the Shadow filter method.
+                for filter_method in SHADOW_FILTER_METHOD.keys():
+                    light_component.set_component_property_value(
+                        AtomComponentProperties.light('Shadow filter method'), SHADOW_FILTER_METHOD[filter_method])
+                    current_filter_method = light_component.get_component_property_value(
+                        AtomComponentProperties.light('Shadow filter method'))
+                    test_shadow_filter_method = (
+                        f"Shadow filter method set to {filter_method}",
+                        f"P1: Shadow filter method set to {filter_method}")
+                    Report.result(test_shadow_filter_method,
+                                  current_filter_method == SHADOW_FILTER_METHOD[filter_method])
+
+                    # 18. Edit the Filtering sample count parameter.
+                    if current_filter_method in (1, 3):
+                        light_component.set_component_property_value(
+                            AtomComponentProperties.light('Filtering sample count'), 64)
+                        Report.result(Tests.edit_filtering_sample_count,
+                                      light_component.get_component_property_value(
+                                          AtomComponentProperties.light('Filtering sample count')) == 64)
+
+                    # 19. Edit the ESM Exponent parameter.
+                    if current_filter_method in (2, 3):
```

Comment:
if filter_method in ['second key', 'third key']

Reply:
Updated to use light_type in ('spot_disk', 'capsule')

Reply:
updated to filter_method in ('ESM', 'PCF+ESM')